evap/staff/templates/staff_semester_view.html

```diff
@@ -37,7 +92,21 @@ <h3>
             <div class="collapse{% if semester.participations_are_archived or semester.grade_documents_are_deleted or semester.results_are_archived %} show{% endif %}" id="archivingCard">
                 <div class="card-body">
                     {% if semester.participations_can_be_archived %}
-                        <button type="button" onclick="archiveParticipationsModalShow({{ semester.id }}, '{{ semester.name|escapejs }}');" class="btn btn-sm btn-warning">{% trans 'Archive participations' %}</button>
+                        <form reload-on-success method="POST" action="{% url 'staff:semester_archive_participations' %}">
```

Comment:
Formatting is broken for the archive buttons
![image](https://github.com/e-valuation/EvaP/assets/1781719/5c8ac7db-aaf1-452c-9083-6bff9fabf17c)